libcxx/docs/Status/Cxx23Papers.csv

```diff
@@ -35,7 +35,7 @@
 "`P2301R1 <https://wg21.link/P2301R1>`__","Add a ``pmr`` alias for ``std::stacktrace``","2021-10 (Virtual)","","",""
 "`P2321R2 <https://wg21.link/P2321R2>`__","``zip``","2021-10 (Virtual)","|In Progress|","",""
 "`P2340R1 <https://wg21.link/P2340R1>`__","Clarifying the status of the 'C headers'","2021-10 (Virtual)","|Nothing To Do|","",""
-"`P2393R1 <https://wg21.link/P2393R1>`__","Cleaning up ``integer``-class types","2021-10 (Virtual)","","",""
+"`P2393R1 <https://wg21.link/P2393R1>`__","Cleaning up ``integer``-class types","2021-10 (Virtual)","|Complete|","21",""
```

Comment:
I believe this paper should be a DR. But the meeting minutes ([N4898](https://wg21.link/n4898)) didn't say so, even thought it said some other LWG papers were DR, which was weird.

Do we want to additional say "this is implemented as a DR against C++20, MSVC STL does the same"? @timsong-cpp @ldionne 

Also, now `libcxx/docs/ReleaseNotes/21.rst` needs to be changed.